src/System.Runtime.Loader/ref/System.Runtime.Loader.csproj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="System.Reflection.Metadata.cs" />
```

Comment:
I thought the name of the file corresponds to the namespace. I guess this is a rare case where the namespace name is different from the library name, I didn't find another case of this.